src/constants.ts

```diff
@@ -5,14 +5,32 @@ import { isNullOrUndefined } from "./util";
 const { pusher, repository } = github.context.payload;
 
 export const workspace: any = process.env.GITHUB_WORKSPACE;
-export const folder = getInput("FOLDER", { required: true });
+export const build = getInput("FOLDER", { required: true });
```

Comment:
These variables need to be set in `main.ts` (or set in a function that is called in `main.ts`), otherwise I still still need to set the environment variables.

Reply:
Ah good point, these are now part of the configuration so you can override them.